models/DataObject/ClassDefinition/Data/Geobounds.php

```diff
@@ -203,7 +203,7 @@ public function getDataForGrid($data, $object = null, $params = [])
      */
     public function getDataFromEditmode($data, $object = null, $params = [])
     {
-        if ($data['NElongitude'] !== null && $data['NElatitude'] !== null && $data['SWlongitude'] !== null && $data['SWlatitude'] !== null) {
+        if ($data && $data['NElongitude'] !== null && $data['NElatitude'] !== null && $data['SWlongitude'] !== null && $data['SWlatitude'] !== null) {
```

Comment:
```suggestion
        if (is_array($data) && $data['NElongitude'] !== null && $data['NElatitude'] !== null && $data['SWlongitude'] !== null && $data['SWlatitude'] !== null) {
```
consistent with [11.0 changes](https://github.com/pimcore/pimcore/pull/14096/files#diff-fbffa0aa1c1ce093e8fb16cfce92d40aed7dd9f167d8481adbc8a696ef9bc9d2R211)